package.json

```diff
@@ -21,7 +21,7 @@
     "build-compare": "webpack --config ./compare/webpack.config.js",
     "dev-compare": "webpack-dev-server --content-base ./compare/output --config ./compare/webpack.config.js",
     "integration-test": "rm -rf newdir && mkdir newdir && cd newdir && node ../cli/index.js genConfig && node ../cli/index.js reference && node ../cli/index.js test && node -e \"require('../')('test')\"",
-    "smoke-test": "cd test/configs/; node ../../cli/index.js test --config=backstop_features",
+    "smoke-test": "node test/smoke/smoke_test.js",
```

Comment:
Very nice harness you built there!  

A suggestion: would you consider leaving the `smoke-test` command the same (so it executes the backstop_features test like before) -- and add a *new* command for your new harness.  Maybe call it like, `travis-test` or `git-commit-test` or `deploy-test`? 

I am totally open to suggestions -- my instinct is to keep it convenient for devs to still be able to run the raw smoke-test and see results (passing or non-passing). 

Does this make sense?

Reply:
I think the harness works well for devs too because it tells them if it succeeds at the end. Right now it's hard to notice if something breaks (e.g., incorrect # of comparison failures or that a timeout). There are some scary looking messages in the output that I don't know if they mean somethings wrong or not (selector not found, etc).

The only difference is that the CI test doesn't pop up the browser report at the end. 

But I will add an argument so if you do
npm run smoke-test 

It will behave just like today with additional niceness and popup the browser, and 

npm run smoke-test -- ci

Will do it without the browser report.

Question for you: Will this have to run in Docker? It seems like launching in travis doesn't really work. If so I can't currently see the docker cloud config - maybe you can share it with me?

Reply:
Sounds good!

I honestly am not an expert in docker -- I just know enough to get me into trouble.  I am happy to share the configs with you -- they should all be in the `/docker` directory.

Also -- just so I am understanding correctly -- when you say...
> It seems like launching in travis doesn't really work
...are you saying you cant run backstop tests in travis?

Reply:
> It seems like launching in travis doesn't really work

Well that's the way it is now - you can click below and see the error. I wasn't sure if you already knew this. If not - no worries I think I can fix this problem. So expect an updated PR shortly.